SmartcarAuth/SmartcarAuth.swift

```diff
@@ -45,15 +45,15 @@ Smartcar Authentication SDK for iOS written in Swift 3.
         - clientId: app client id
         - redirectUri: app redirect uri
         - scope: app oauth scope
-        - development: optional, shows the mock OEM for testing, defaults to false
+        - testMode: optional, launch the Smartcar auth flow in test mode, defaults to false
         - completion: callback function called upon the completion of the OAuth flow with the error, the auth code, and the state string
     */
-    @objc public init(clientId: String, redirectUri: String, scope: [String] = [], development: Bool = false, completion: @escaping (Error?, String?, String?) -> Any?) {
+    @objc public init(clientId: String, redirectUri: String, scope: [String] = [], testMode: Bool = false, completion: @escaping (Error?, String?, String?) -> Any?) {
```

Comment:
This was a breaking change since the user needs to specify the name of the parameter. @jerzzhang is managing either unpublishing this version from cocoapods and putting up a new version that is backwards compatible with development or depcreating this version and properly publishing a breaking change. This should not have been a breaking change.

Reply:
cc: @esonmez